compiler/rustc_target/src/spec/targets/i686_unknown_linux_gnu.rs

```diff
@@ -1,7 +1,12 @@
-use crate::spec::{Cc, LinkerFlavor, Lld, SanitizerSet, StackProbeType, Target, base};
+use crate::spec::{Cc, LinkerFlavor, Lld, RustAbi, SanitizerSet, StackProbeType, Target, base};
 
 pub(crate) fn target() -> Target {
     let mut base = base::linux_gnu::opts();
+    base.rust_abi = Some(RustAbi::X86Sse2);
```

Comment:
if this doesn't affect only the `extern "Rust"` ABI then it has to not be this name.